src/Flag.ts

```diff
@@ -11,9 +12,15 @@ class Flag extends UiComponent {
    */
   public static readonly googleLimiter = new TokenBucketLimiter({ bucketSize: 1, tokensPerInterval: 1, interval: 1000 * 2, stopped: true })
   public static readonly cloudFlareLimiter = new TokenBucketLimiter({ bucketSize: 1, tokensPerInterval: 1, interval: 1000 * 2, stopped: true })
+  private readonly ipfsClient
 
   constructor (protected parent: GatewayNode, private readonly hostname: string) {
     super(parent, 'div', 'Flag')
+    this.ipfsClient = create({
+      host: 'ipfs.io',
+      port: 443,
+      protocol: 'https'
+    })
```

Comment:
we don't need a window level object.

Reply:
while we don't necessarily want global objects, I think setting up the client to exist inside of flag is the wrong path forward because we most likely need to use the client elsewhere, yes? Should we have a client created that we can use across the different components/checks?

Reply:
@SgtPooki I could not find reference to `window.IpfsClient` it looks like it's a remanent from migration to typescript.